homeassistant/components/influxdb.py

```diff
@@ -52,6 +53,7 @@ def setup(hass, config):
     ssl = util.convert(conf.get(CONF_SSL), bool, DEFAULT_SSL)
     verify_ssl = util.convert(conf.get(CONF_VERIFY_SSL), bool,
                               DEFAULT_VERIFY_SSL)
+    blacklist = conf.get('blacklist', [])
```

Comment:
OK, I see it now. Ishould have wait for a moment with more time ahead to review your comments.